README.md

```diff
@@ -1,39 +1,60 @@
 [![Build Status](https://travis-ci.org/transifex/transifex-python.svg?branch=master)](https://travis-ci.org/transifex/transifex-python)
 [![codecov](https://codecov.io/gh/transifex/transifex-python/branch/master/graph/badge.svg)](https://codecov.io/gh/transifex/transifex-python)
 
-Transifex Python Toolkit
-------------------------
+# Transifex Python Toolkit
 
-Transifex Python Toolkit is a collection of tools that allow you to easily localize your applications using Transifex.
+Transifex Python Toolkit (referred as **the Toolkit** for brevity) is a collection of tools that allow you to easily localize your applications using Transifex (TX). The toolkit features fetching translations over the air (OTA) to your apps.
+
+Using Transifex (TX) the localization workflow typically consists of the following steps:
+1. Find - extract the source strings in your source code
+2. Push them into a Transifex project
+3. In Transifex your source strings get translated by translators in target languages
+4. Retrieve a string translation of a target language from Transifex to display it
+
+Transifex Python Toolkit is used in steps 1, 2, 4 above. Possible usage scenarios:
+* [In a Django web application](#scenario-1-integrate-with-a-django-app) The toolkit is fully integrated in Django framework. It contains utilities to mark your source strings for translation and display translation in templates or views. It also provides special CLI commands for pushing to TX and even migrating your existing template code to the new syntax.
```

Comment:
Let's add the bold
...to mark your source strings for translation, **retrieve the translated strings** and display translation**s** in templates or views.